src/Runtime/Factors/StringFormatOp.cs

```diff
@@ -681,7 +683,7 @@ public class StringFormatOpBase_NoArgumentNames<TStringFormatOp>
         /// <summary>
         /// Maps the number of argument to an array of default argument names ("0", "1" and so on).
         /// </summary>
-        private static readonly Dictionary<int, string[]> ArgumentCountToNames = new Dictionary<int, string[]>();
+        private static ReadOnlyArray<string>?[] ArgumentCountToNames = new ReadOnlyArray<string>?[0];
```

Comment:
ReadOnlyArray can be null. It is checked through "IsNull" method. Get rid of the question mark. Also, increase initial cache size to 16